### PR TITLE
ec_deployment: Note on importing with user settings (config)

### PR DESCRIPTION
**Pull request by karencfv**
## Description
Note on support regarding importing deployments with user settings.

### CODE REVIEW

File: docs/resources/ec_deployment.md [hunk 1]
```diff
@@ -317,6 +317,8 @@ In addition to all the arguments above, the following attributes are exported:
 
 ~> **Note on pre 6.6.0 deployments** Importing deployments with a version lower than `6.6.0` is not supported.
 
+~> **Note on deployments with per topology user settings** Only deployments with global user settings (config) are supported. Make sure to migrate to global settings before importing.
```
Comment (alaudazzi) [inline docs/resources/ec_deployment.md:320]:
```suggestion
~> **Note on deployments with topology user settings** Only deployments with global user settings (config) are supported. Make sure to migrate to global settings before importing.
```

Comment (alaudazzi) [reply docs/resources/ec_deployment.md:320]: Or as an alternative
topology-based settings

**Review by alaudazzi — APPROVED** [review]
Minor editing suggestion. LGTM.

**Review by marclop — APPROVED** [review]
LGTM

**Review by nrichers — APPROVED** [review]
LGTM 🚀

**Pull request merged**